db/migrate/20231201091435_create_site_tags.rb

```diff
@@ -0,0 +1,293 @@
+class CreateSiteTags < ActiveRecord::Migration[7.0]
+  def change
+    return if Rails.env.production?
+
+    create_table :site_tags do |t|
+      t.string :tag, null: false, index: { unique: true }
+      t.string :description, null: false
+
+      t.timestamps
+    end
+
+    [
```

Comment:
Let me know if you don't want this in the migration. It is an easy way for us to get the data locally 🤷

Reply:
It's fine.